corehq/ex-submodules/couchexport/export.py

```diff
@@ -309,7 +309,8 @@ def split_path(path):
         id = []
         for k in path:
             if isinstance(k, basestring):
-                column.append(k)
+                if k:
+                    column.append(k)
```

Comment:
i actually think we specifically add `''` as a column. the situation arises when you have a repeat group with id `question1` and then delete it and add another question of the same id (say `question1`) and it is a text question o something else. when you click show deleted, `question1.` will appear and that will be your way to export a repeat group. not the greatest